CHANGELOG.rst

```diff
@@ -49,6 +49,10 @@ in development
   put the workflow in a PAUSED state in mistral. (improvement)
 * Add support for evaluating jinja expressions in mistral workflow definition where yaql
   expressions are typically accepted. (improvement)
+# Added support for `st2 login` and `st2 whoami` commands. These add some additional functionality
```

Comment:
Doh! Fixed in https://github.com/StackStorm/st2/pull/3123/commits/3da2b42f05de1fc1c922a5a5c01c8bbb3088feff